lib/menu.rb

```diff
@@ -0,0 +1,29 @@
+class Menu
+
+	attr_reader :menu_list, :menu_keys, :menu_values
+
+	def initialize
+		@menu_list = menu_list
+		@menu_keys = menu_list.keys
+		@menu_values = menu_list.values
```

Comment:
I don't think these need to be split into their own instance variables. You could use a private method to fetch these from `@menu_list`, and then limit the number of instance variables this class has.